README.md

```diff
@@ -135,12 +135,12 @@ There are totally 19 workloads in HiBench. The workloads are divided into 6 cate
 4. Fixwindow (fixwindow)
 
     The workloads performs a window based aggregation. It tests the performance of window operation in the streaming frameworks.
-  
-    
-### Supported Hadoop/Spark/Flink/Storm/Gearpump releases: ###
 
-  - Hadoop: Apache Hadoop 2.x, CDH5, HDP
-  - Spark: Spark 1.6.x, Spark 2.0.x, Spark 2.1.x, Spark 2.2.x
+### Supported Hadoop/Spark releases: ###
+  - Hadoop: Apache Hadoop 2.x, 3.x, CDH5, HDP
```

Comment:
**Q:** Did you test Hadoop 3.0, 3.1, 3.3?
**A:** No, you right, Otherwise 2.x, 3.2.

**Q:** Why do you separate streaming/non-streaming frameworks？
**A:** Because Scala < 2.12 does not compiles on java 1.11 jdk and, scala 2.12
requires to change (or bump) the package org.apache.kafka **from** `0.8.2.1` **to at least** `0.10.2.2` and then the whole code related with Kafka and streaming testing must be ported.
This last kafka version (0.10.2.2) will require to modify following classes:
    
* KafkaCollector.scala
* KafkaConsumer.scala
* MetricsUtil.scala

So, bottom line, as mentioned in previous [comment](https://github.com/Intel-bigdata/HiBench/pull/586#discussion_r298683334) for @carsonwang, to avoid break the streaming benchmarks in scala 2.11 and 2.10 was streaming/non-streaming frameworks split.

Reply:
Done: **Otherwise 2.x, 3.2.**